docs/src/tutorials/beginner_tutorial.md

```diff
@@ -0,0 +1,229 @@
+
+To use the `DifferentialEquations` library, we need to define a differential equation as a function, solve it and then plot it (we know how to do the last part).
+
+Another useful library we'll be using is `ModelingToolkit.jl`, a Julia package designed to make our lives easiear when dealing with complex models and performing tasks such as model simplification, parameter estimation and code generation.
+
+Let's solve the ODE $\frac{du}{dt}=-a \cdot u$  using this package. First we need to define what are our variables and what are our parameters. For this, we'll use the `@variables`and `@parameters` macros from `ModelingToolkit`:
+
+```@example beginner_tutorial
+# Install and import these packages
+using Pkg
+Pkg.add("ModelingToolkit")
+Pkg.add("DifferentialEquations")
+using ModelingToolkit
+using DifferentialEquations
+
+# Define the time variable
+@variables t
+
+# Define the time-dependent variable u
+@variables u(t)
+
+# Define the parameter a
+@parameters a
+```
+
+We also have to define our differential operator (it'll work as a function). We just need to declare it using the function `Differential(independent_variable)`:
+
+```@example beginner_tutorial
+# Define the differential operator 
+D = Differential(t)
+```
+
+Finally, when using `ModelingToolkit`, the `~` symbol is used as the equal sign when writing an equation:
+
+```@example beginner_tutorial
+# Define the differential equation du/dt = -a * u
+equation = D(u) ~ -a * u
+```
+
+Now, in order to build the ODE system, we use the `@named` macro to name the system and the `ODESystem` function:
+
+```@example beginner_tutorial
+# Build the ODE system and assign it a name
+@named system = ODESystem(equation, t)
+```
+
+We also have to define our system's initial conditions and parameter values. We use the following syntax:
+
+```@example beginner_tutorial
+# Define the initial conditions and parameters
+u0 = [u => 1.0]
+p = [a => 2.0]
+tspan = (0.0 , 5.0)
+```
+
+Finally, using the functions `ODEProblem` and `solve` from `DifferentialEquations` we convert our system to a numerical problem and solve it:
+
+```@example beginner_tutorial
+# Set up the problem
+problem = ODEProblem(complete(system), u0, tspan, p)
+
+# Solve the problem
+solution = solve(problem)
+```
+
+Finally, we can graph the solution using what we've already learned:
+
+```@example beginner_tutorial
+using Plots
+
+plot(solution, xlabel="Time", ylabel="u(t)", title="Solution to du/dt=-a*u")
+```
+
+## The Izhikevich model
+
+The Izhikevich model is described by the following two-dimensional system of ordinary differential equations:
+
+```math
+\begin{align}
+\frac{dv}{dt} &= 0.04 v^2 + 5 v + 140 - u + I \\
+\frac{du}{dt} &= a(bv - u)
+\end{align}
+```
+
+with the auxiliary after-spike resetting
+
+```math
+\begin{align}
+\text{if }v \geq 30\text{ mV, then }\begin{cases}v\leftarrow c, u\leftarrow u + d.\end{cases}
+\end{align}
```

Comment:
```suggestion
\begin{align}
\text{if }v \geq 30\text{ mV, then }\begin{cases}v\rightarrow c\\ u \rightarrow u + d \end{cases}
\end{align}
```
maybe a little nicer to have this as two lines, and I assume you meant to have `\rightarrow` on that.